context/entrypoint.sh

```diff
@@ -17,5 +17,18 @@ source /opt/docker/bin/packages.sh
 SRC_FILE="/opt/docker/bin/entrypoint_source"
 [ -f "${SRC_FILE}" ] && source "${SRC_FILE}"
 
+# Check if we should quote the exec params
+UNQUOTE=false
+if [ "$1" = "--unquote-exec" ]; then
```

Comment:
This change is to support where `$@` is already a quoted string. So I'm not sure whether `$1` will actually be `"--unquote-exec"`. It might be `"\"--unquote-exec"`, but I'm honestly not sure.

Reply:
I pushed a new version that should work in that case. WDYT?

Reply:
Thansk @Ethyling. That makes sense, but I also pinged @sriramch who seems happy that the original fix should also work.

Reply:
Oh ok, so I think I will revert to the previous version then. Thank you!

Reply:
Done